source/GameData.cpp

```diff
@@ -1202,6 +1211,8 @@ void GameData::LoadFile(const string &path, bool debugMode)
 				text += child.Token(0);
 			}
 		}
+		else if(key == "substitutions" && node.HasChildren())
+			substitutions.Load(node);
```

Comment:
This is needed too in `GameData::Change` or else events can't change the global substitutions.